miniwin/src/d3drm/backends/opengles3/renderer.cpp

```diff
@@ -1,8 +1,8 @@
-#include "d3drmrenderer_opengles2.h"
+#include "d3drmrenderer_opengles3.h"
 #include "meshutils.h"
 
-#include <GLES2/gl2.h>
 #include <GLES2/gl2ext.h>
```

Comment:
```suggestion
```

Reply:
Can't remove it because `GL_TEXTURE_MAX_ANISOTROPY_EXT` is defined in it

Reply:
yeah your right :)